setup.py

```diff
@@ -16,7 +16,7 @@
 
 
 setup(name='PyU4V',
-      version='9.1.0.0',
+      version='9.1.1.0',
```

Comment:
Does PyU4V.tools need to be added to packages in the same manner as PyU4V.utils?